.github/workflows/ci_wontrun.yml

```diff
@@ -0,0 +1,27 @@
+name: CodeBuild
+on:
+  pull_request:
+    branches: [main]
+  merge_group:
+    types: [checks_requested]
+    branches: [main]
+jobs:
+  buildspec:
+    runs-on: ubuntu-latest
+    name: Check for modified spec files
+    steps:
+      - users: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get altered spec files
+        id: spec-files-changed
+        uses: tj-actions/changed-files@v35
+        with:
+          files: codebuild/spec/*.{yml,yaml}
```

Comment:
Any way we can make sure this matches the actual pattern Codebuild uses?

Reply:
Good point! codebuild jobs by default check for `/buildspec.yml` so that needs to also be in there.